minimum_purchase_order_value_per_vendor/views/res_partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+    <record id="view_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.form.block.reason</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[@name='sales_purchases']//group[@name='purchase']//field[@name='supplier']"
+                   position="after">
+                <field name="minimum_po_amount" attrs="{'invisible': [('supplier', '=', False)]}" />
```

Comment:
respect margin

Reply:
Don't you think it is fine, as I using that field in xpath.

Reply:
ok

Reply:
Is it `minimum quantity`? or `minimum PO amount` ?

Reply:
yes, the minimum PO amount per vendor.

Reply:
Okay.